File: has23/Has23.java
package has23;

public class Has23 {
    //Dado um Array de "length" 2, retorne true se ele tiver um 2 ou um 3.

    public boolean has23(int[] nums) {
        for (int n :
                nums) {
            if (n == 2 || n == 3) return true;
        }
        return false;
    }
}

File: Python/List-1/has23.py
"""
Dada uma lista numérica "nums" de tamanho 2, retorne true se ela possuir uma valor
"2" ou "3".
"""


def has23(nums):
    return nums.count(2) > 0 or nums.count(3) > 0


print(has23([1, 4]))
